viewer/packages/pointclouds/src/potree-three-loader/Potree.ts

```diff
@@ -117,96 +127,143 @@ export class Potree implements IPotree {
     return EptBinaryLoader.WORKER_POOL.maxWorkers;
   }
 
+  private updateVisibilityForNode(
```

Comment:
What is this change?

Reply:
I factored out this function from the `updateVisibility` function (commit [6ad41ca](https://github.com/cognitedata/reveal/pull/2193/commits/6ad41ca68399edbf88831169e7ab0e31e75c5209)) and then inserted another call to it to ensure it is called at least for the root node (commit [6bcf1ea](https://github.com/cognitedata/reveal/pull/2193/commits/6bcf1ea98b6b6140f64ea5c5fa381b8bf44e8967)). It fixed the low-budget picking issue.